libathemecore/account.c

```diff
@@ -371,7 +371,7 @@ myuser_rename(struct myuser *mu, const char *name)
 	entity(mu)->name = newname;
 
 	myentity_put(entity(mu));
-	if (authservice_loaded)
+	if (authservice_loaded && !(mu->flags & MU_WAITAUTH))
```

Comment:
hrm, i'm not sure the `MU_WAITAUTH` check here ought to cover the hook call as well?

Reply:
(nevermind, misread)